docs/Documentation/torch_compile.md

```diff
@@ -0,0 +1,157 @@
+# PyTorch Deployment via "torch.compile" {#pytorch_2_0_torch_compile}
+
+@sphinxdirective
+
+
+The ``torch.compile`` feature enables you to use OpenVINO for PyTorch-native applications. 
+It speeds up PyTorch code by JIT-compiling it into optimized kernels.
+By default, Torch code runs in eager-mode, but with the use of ``torch.compile`` it goes through the following steps:
+
+1. **Graph acquisition** - the model is rewritten as blocks of subgraphs that are either:
+
+   * compiled by TorchDynamo and "flattened",
+   * falling back to the eager-mode, due to unsupported Python constructs (like control-flow code).
+
+2. **Graph lowering** - all PyTorch operations are decomposed into their constituent kernels specific to the chosen backend.
+3. **Graph compilation** - the kernels call their corresponding low-level device-specific operations.
+
+
+
+How to Use
+#################
+
+To use ``torch.compile``, you need to add an import statement and define one of the two available backends:
+
+| ``openvino``
+|   With this backend, Torch FX subgraphs are directly converted to OpenVINO representation without any additional PyTorch based tracing/scripting.
+
+| ``openvino_ts``
+|   With this backend, Torch FX subgraphs are first traced/scripted with PyTorch Torchscript, and then converted to OpenVINO representation.
+
+
+.. tab-set::
+
+   .. tab-item:: openvino
+      :sync: backend-openvino
+
+      .. code-block:: console
+
+         import openvino.torch 
+         ...
+         model = torch.compile(model, backend='openvino')
+
+      Execution diagram:
+
+      .. image:: _static/images/torch_compile_backend_openvino.svg
+         :width: 992px
+         :height: 720px
+         :scale: 60%
+         :align: center
+
+   .. tab-item:: openvino_ts
+      :sync: backend-openvino-ts
+
+      .. code-block:: console
+
+         import openvino.torch
+         ...
+         model = torch.compile(model, backend='openvino_ts')
+
+      Execution diagram:
+
+      .. image:: _static/images/torch_compile_backend_openvino_ts.svg
+         :width: 1088px
+         :height: 720px
+         :scale: 60%
+         :align: center
+
```

Comment:
@suryasidd have not we mixed description and execution diagram ?
<img width="802" alt="Screenshot 2023-09-08 at 4 12 15 PM" src="https://github.com/openvinotoolkit/openvino/assets/2566854/3ef364dd-cbb2-4730-9aea-679b40a8729d">

Reply:
Yes, both openvino and openvino_ts diagrams are out of place. Thanks for catching that. Fixes:

https://github.com/openvinotoolkit/openvino/pull/19699
https://github.com/openvinotoolkit/openvino/pull/19700
https://github.com/openvinotoolkit/openvino/pull/19701